config/standard/tasks.js

```diff
@@ -322,21 +322,85 @@
     },
   },
 
-  // create nutrition screening task if degree of severity is moderate/severe
+  // followup tasks as per nutrition program schedule (OTP, SFP, or SC)
+  {
+    icon: 'child',
+    title: 'task.nutrition_followup.title',
+    appliesTo: 'scheduled_tasks',
+    appliesToType: ['nutrition_screening'],
+    appliesIf: function(c, r) {
+      return (
+        isCoveredByUseCase(c.contact, 'gmp') &&
+        r.fields.treatment.program &&
+        (r.fields.treatment.program === 'OTP' || r.fields.treatment.program === 'SFP' || r.fields.treatment.program === 'SC')
+      );
+    },
+    actions: [{ form: 'nutrition_followup' }],
+    events: [
+      {
+        id: 'nutrition-followup-missing-visit',
+        dueDate: function(r, e, i){
+          return new Date(r.scheduled_tasks[i].due.slice(0, 10));
+        },
+        start: 0,
+        end: 3,
+      }
+    ],
+    resolvedIf: function(c, r, event, dueDate, i) {
+      return (
+        r.scheduled_tasks[i].state === 'cleared' ||
+        isFormFromArraySubmittedInWindow(
+          c.reports,
+          ['nutrition_followup', 'CF'],
+          Utils.addDate(dueDate, 0).getTime(),
+          Utils.addDate(dueDate, event.end + 1).getTime()
+        )
+      );
+    }
+  },
+
+  // create nutrition screening task if degree of severity is moderate or severe
   {
     icon: 'child',
     title: 'task.nutrition_screening.title',
     appliesTo: 'reports',
     appliesToType: ['G'],
     appliesIf: function(c, r, i){
       /* jshint unused:vars */
-      return r.fields.severity === "2" || r.fields.severity === "3";
+      return r.fields.severity === "3" || r.fields.severity === "2";
```

Comment:
Curious what the problem was here. Also, is there a chance that this field is an integer and not a string representation of the integer?

Reply:
This wasn't intentional, the change came when the task was removed and added back in later. Severity can be an integer if configured as such, getting around this by always casting to string.